src/steps/generic.rs

```diff
@@ -318,6 +318,27 @@ pub fn run_vcpkg_update(ctx: &ExecutionContext) -> Result<()> {
     command.args(["upgrade", "--no-dry-run"]).status_checked()
 }
 
+pub fn run_vscode_extensions_upgrade(ctx: &ExecutionContext) -> Result<()> {
+    let vscode = require("code")?;
+    print_separator("Visual Studio Code extensions");
+
+    let extensions = Command::new(&vscode)
+        .arg("--list-extensions")
+        .output_checked_utf8()?
+        .stdout;
+
+    if !extensions.is_empty() {
```

Comment:
Since an update command is currently unavailable, would you like to add some comments here stating the approach we use and linking that VSCode issue:)

Reply:
Yes, for sure! I just pushed a commit to comment that ;)